docs/reverse_proxy.md

```diff
@@ -134,6 +137,8 @@ example.com:8448 {
 ```
 frontend https
   bind :::443 v4v6 ssl crt /etc/ssl/haproxy/ strict-sni alpn h2,http/1.1
+  http-request add-header X-Forwarded-Proto https if { ssl_fc }
```

Comment:
Should this also be `set-header`?

From the [haproxy docs](https://www.haproxy.com/documentation/aloha/latest/traffic-management/lb-layer7/http-rewrite/):

> `set-header` removes first any header in existing field that matches `<name>`.

Reply:
yes.

I've also added an equivalent for `http`, which is strictly redundant if people only configure an `ssl` bind, but I figure that it might be important if it gets cut-and-pasted.